TheAurorian/src/main/resources/assets/theaurorian/patchouli_books/the_aurorian_guide/zh_cn/entries/magicalitems/absorptionorb.json

```
@@ -0,0 +1,13 @@
+{
+    "name": "修损宝珠",
+    "icon": "theaurorian:absorptionorb",
+    "category": "3magicalitems",
+    "readbydefault": true,
+    "pages": [
+        {
+            "type": "spotlight",
+            "item": "theaurorian:absorptionorb",
+            "text": "$(#008cff)修损宝珠$()可以在在副手时恢复你主手物品的耐久度！"
```

|Feedback Priority|
|---|
|🧗‍♀️ Boulder / **Blocking**|

```suggestion
            "text": "$(#008cff)修损宝珠$()可以恢复你主手物品的耐久度！"
```
删掉的这句话需要实测。